src/services/user/index.js

```diff
@@ -59,6 +59,12 @@ module.exports = (app) => {
 	const firstLoginService = app.service('firstLogin');
 	firstLoginService.hooks(firstLoginHooks);
 
+	// eslint-disable-next-line global-require
+	const ForcePasswordChangeService = require('./services/forcePasswordChange')(app);
```

Comment:
please do not add require on this point it is not necessary.

Reply:
Ok, so where should I put it ? As I have already mentioned it cannot be putted on top
https://github.com/schul-cloud/schulcloud-server/pull/1455#discussion_r446817926

Reply:
I will check it in code it must go in one or other way.